bin/assemble_fibermap

```diff
@@ -16,6 +16,8 @@ parser.add_argument("-o", "--outfile", type=str, required=True,
 parser.add_argument("-b","--badamps", type=str,
         help="comma separated list of {camera}{petal}{amp}"+\
              ", i.e. [brz][0-9][ABCD]. Example: 'b7D,z8A'")
+parser.add_argument("--badfibers", type=str,
+        help="filename with table of bad fibers (with at least FIBER and FIBERSTATUS columns)")
```

Comment:
For the record: this PR is no longer using the `assemble_fibermap --badfibers ...` option in lieu of collecting bad *columns* and masking out wavelength ranges instead of automatically flagging entire fibers.  But we'll leave this option in here in case we want to use it for the future.